src/ArrListLinkedList/ArrayList.java

```diff
@@ -0,0 +1,65 @@
+package ArrListLinkedList;
```

Comment:
bad name for package
https://mate-academy.github.io/style-guides/java/java.html#s5.2.1-package-names